Reword the watcher 'always' and 'never' condition docs (#86105)
elastic/elasticsearch: x-pack/docs/en/watcher/condition/always.asciidoc

<titleabbrev>Always condition</titleabbrev>
++++

> Use the `always` condition so that watch actions will always be
> executed when the watch is triggered, unless they are
> <<actions-ack-throttle,throttled>>.

Reviewer: ```suggestion
Use the `always` condition to perform the watch actions whenever the watch is triggered, unless they are
<<actions-ack-throttle,throttled>>.
```
